src/Admin/Menu/AbstractMenuNodeAdmin.php

```diff
@@ -34,6 +34,11 @@
     protected $menuRoot;
 
     /**
+     * @var bool
+     */
+    protected $useBurgovKeyValueForm;
```

Comment:
why is this on the base class? it is only used in one place, lets move it to the specific class until we need it in more places.